crates/driver/src/infra/config/file/mod.rs

```diff
@@ -574,3 +579,19 @@ pub enum GasEstimatorType {
     Native,
     Web3,
 }
+
+#[derive(Debug, Deserialize)]
+#[serde(rename_all = "kebab-case", deny_unknown_fields)]
+pub enum OrderPriorityStrategy {
```

Comment:
This needs some serious documentation if we expect people to configure this (especially on how a list of different strategies is supposed to work).

Reply:
https://github.com/cowprotocol/services/pull/2872